README.md

```diff
@@ -1,13 +1,21 @@
 # DeepSea
 A collection of Salt files for deploying, managing and automating Ceph.
 
-The goal is to manage multiple Ceph clusters with a single salt master.  At this time, only a single Ceph cluster can be managed.
+The goal is to manage multiple Ceph clusters with a single salt master. At this time, only a single Ceph cluster can be managed.
 
 The [diagram](deepsea.png) should explain the intended flow for the orchestration runners and related salt states.
 
 ## Status
-Automatic discovery, configuration and deployment of Ceph clusters works. RGW
-deployment works for single site deployements. MDS deployment and CephFS creation works.
+DeepSea currently supports the following functionality:
+
+- Automatic discovery, deployment, configuration and life cycle management of Ceph clusters
+- Initial support for importing Ceph clusters deployed using `ceph-deploy`
```

Comment:
I think we are not limited to ceph-deploy. Technically the cluster could've been deployed with any tool